rllab/misc/logger.py

```diff
@@ -198,8 +201,10 @@ def dump_tabular(*args, **kwargs):
         # Also write to the csv files
         # This assumes that the keys in each iteration won't change!
         for tabular_fd in list(_tabular_fds.values()):
-            writer = csv.DictWriter(tabular_fd, fieldnames=list(tabular_dict.keys()))
-            if wh or (wh is None and tabular_fd not in _tabular_header_written):
```

Comment:
Is there a reason this file needed to be modified to get dm_control to work? Otherwise, these changes should be moved into a different PR.

Reply:
If a change is not related to the content of a commit/pull request, it should not appear in that commit/pull request.